example.env

```diff
@@ -84,10 +84,10 @@ OSM_REGISTER_URL=https://www.openstreetmap.org/user/new
 # It's not required to set this tag. Case it isn't set, an image will be used as background.
 # TM_HOMEPAGE_VIDEO_URL=
 
-# Endpoint for the missing maps stats
+# API base URL and token(used to retrieve user stats only) for ohsomeNow Stats
 #
-TM_USER_STATS_API_URL=https://osm-stats-production-api.azurewebsites.net/users/
-TM_HOMEPAGE_STATS_API_URL=https://osmstats-api.hotosm.org/wildcard?key=hotosm-project-*
+OHSOME_STATS_BASE_URL=https://demo.contributions-stats.ohsome.org/api
```

Comment:
I know that this is only the example.env but you could replace the outdated URL.

the new one is https://stats.now.ohsome.org/api as you have also listed below

Reply:
Thank you for pointing this out